### PR TITLE
Adds Refrigeration and Spoiling

### PR DESCRIPTION
**Pull request by redmoogle**
# General Documentation

### Intent of your Pull Request

right on the tin. (although this only apply to meats for now...)

Meat can now spoil if it's not in a freezer or food cart and will turn in to burned mess (or rotten meat for raw meat)
The time in which this happens is 12.5 minutes

## Todo

- [x] Atmospheric Freezing (may not be feasible)
- [x] More things that spoil/refrigerators
- [ ] Get sprites for spoiled things

### Why is this change good for the game?

Interesting dynamic which chef is _slightly_ responsible for his food

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Chefs have to put their dirty meat up

### What should players be aware of when it comes to the changes your PR is implementing?
Meat Express

### What general grouping does this PR fall under? 
Chef Nerf

# Changelog

:cl:  
rscadd: Meat now spoils if it's not in a fridge or a food cart
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
if it was to be implemented with atmos you should add h/e pipes, a freezer, and an n2 canister to kitchens imo

**Comment by Tipy1802** [comment]
Make sure they don't spoil when in the cold room because well it's a cold room

**Comment by Marmio64** [comment]
Maybe turn the burned mess desc and name for rotten items to better reflect that it rotted rather than burned

**Comment by redmoogle** [comment]
> 
> 
> Make sure they don't spoil when in the cold room because well it's a cold room

this may not be feasible but I will try

**Comment by boodaliboo** [comment]
make rotten food make miasma

**Comment by MegaEmpirical** [comment]
Yessss. Make bad meat that's not refrigerated make miasma!

**Comment by JamieD1** [comment]
Test Merge

**Comment by MegaEmpirical** [comment]
Noooooo
